### PR TITLE
chore(package): update deps

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,9 +89,10 @@
     "hyperhtml": "^2.10.0",
     "loading-indicator": "^2.0.0",
     "marked": "^0.3.19",
+    "npm": "^6.0.1",
```
Comment (saschanaz) [inline package.json]: Should we do this, or `"npm": ">=5"` in `engines` field instead?

Comment (saschanaz) [reply package.json]: Or just skip this line? I'm not sure installing npm as a dependency is right here...

Comment (marcoscaceres) [reply package.json]: agree.

**Comment by marcoscaceres** [comment]
package-lock.json has messy merge conflicts... sending again.

**Pull request closed without merging**